agreements/project-contribution/fiscal-sponsorship-amendment-email.md

```diff
@@ -0,0 +1,45 @@
+# Email Template for Amendment to Fiscal Sponsorship Agreement
+
+This email format can be used by a Project Representative or Signatory to request a change to the Fiscal Sponsorship Agreement (FSA).
+
+An amendment is valid if:
+
+- All required parties are included on the thread (e.g. Commonhaus Legal, existing and new signatories)
+- The email clearly states what is being amended
+- Each party replies affirmatively to confirm
+
+Notes:
+
+- `project-[project-name]@commonhaus.org` is the shared group email provided by Commonhaus for communication with project signatories.
+- This form is appropriate for administrative updates like adding or removing a Signatory.
+
+---
+
+## Template
+
+**Subject:** Request to Amend Fiscal Sponsorship Agreement – `[Project Name]`  
+**To:** `legal@commonhaus.org`, `[New or Resigning Signatory]`  
+**Cc:** `project-[project-name]@commonhaus.org`  
+**From:** `[Project Signatory Email]`
+
+---
+
+To all concerned,
+
+On behalf of the “[Project Name]” project, I’m requesting an amendment to the Fiscal Sponsorship Agreement originally signed on `[Original Agreement Date]`.
+
+This amendment reflects the following changes:
+
+1. `[e.g. Add New Signatory – Name, Email]`  
+2. `[e.g. Remove Signatory – Name, Email]`  
```

Comment:
Does the use of a numbered list imply that the transfer of signatories is atomic? The project must maintain 2+ signatories at all times?

Reply:
No, notice the first is Add, the second is remove. Only meant to show that more than one can be included in the same document

Reply:
Ok.

For some reason I can't mark the thread as resolved :/